### PR TITLE
Add an extra disk partition and mounting

### PR DESCRIPTION
**Pull request by jan-est**
`metal3-dev-env` allows an extra disk to be added to hosts using the environment variable` VM_EXTRADISKS`.

This PR adds two new environment variables `VM_EXTRADISKS_FILE_SYSTEM` and` VM_EXTRADISKS_MOUNT_DIR`. These variables allow the user to select the file system to use on the extra disk and specify the path where the extra disk will be installed. With this PR, `metal3-dev-env` can also partition an extra disk.

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/vars/main.yml [hunk 1]
```diff
@@ -19,7 +19,9 @@ IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
 KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.21.0', true) }}"
 MAX_SURGE_VALUE: "{{ lookup('env', 'MAX_SURGE_VALUE') | default('1', true) }}"
-
+VM_EXTRADISKS: "{{ lookup('env', 'VM_EXTRADISKS') | default('false', true) }}"
+VM_EXTRADISKS_FILE_SYSTEM: "{{ lookup('env', 'VM_EXTRADISKS_FILE_SYSTEM') | default('ext4', true) }}"
+VM_EXTRADISKS_MOUNT_DIR: "{{ lookup('env', 'VM_EXTRADISKS_MOUNT_DIR') | default('/mnt/disk2') }}"
```
Comment (furkatgofurov7) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:24]: Maybe
```suggestion
VM_EXTRADISKS_MOUNT_DIR: "{{ lookup('env', 'VM_EXTRADISKS_MOUNT_DIR') | default('/mnt/disk2', true) }}" 
```
?

**Review by furkatgofurov7 — COMMENTED** [review]
Mostly lgtm, question inline

File: vm-setup/roles/v1aX_integration_test/vars/main.yml (hunk 1 above)
Comment (jan-est) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml:24]: I don't know if it's necessary to use . see line 61 `IRONIC_IMAGE_DIR: "{{ lookup('env', 'IRONIC_IMAGE_DIR') | default('/opt/metal3-dev-env/ironic/html/images')}}"`

**Review by kashifest — APPROVED** [review]

**Comment by kashifest** [comment]
It needs rebase @jan-est

**Comment by jan-est** [comment]
> It needs rebase @jan-est

Done!

**Comment by maelk** [comment]
Could you please add a bit more info in this PR message about the goal. What are you using this feature for, from a user point of view ?

**Comment by jan-est** [comment]
> Could you please add a bit more info in this PR message about the goal. What are you using this feature for, from a user point of view ?

Done!

**Review by furkatgofurov7 — COMMENTED** [review]
This should be already good to go in. 

/lgtm

**Pull request merged**